src/constants.js

```diff
@@ -6,5 +6,8 @@ export default {
     copy: {
```

Comment:
All I did was copy the existing inline string into the constants file for consistency.